at.priv.toastfreeware.ConfClerk.yaml

```
@@ -0,0 +1,45 @@
+app-id: at.priv.toastfreeware.ConfClerk
+runtime: org.kde.Platform
+runtime-version: '5.12'
+sdk: org.kde.Sdk
+command: confclerk
+rename-icon: confclerk
+copy-icon: confclerk
```

```suggestion
copy-icon: true
```
it's boolean to indicate whether to copy the icon under the new name or move it if it's set to false.